crates/executor/src/executor.rs

```diff
@@ -254,8 +260,11 @@ pub fn execute_and_verify_receipt<DB: StateProvider>(
 ) -> Result<ExecutionResult, Error> {
     let transaction_change_set = execute(header, transactions, config, db)?;
 
-    let receipts_iter = transaction_change_set.changeset.iter().map(|changeset| &changeset.receipt);
-    verify_receipt(header.receipts_root, header.logs_bloom, receipts_iter)?;
+    let _receipts_iter =
+        transaction_change_set.changesets.iter().map(|changeset| &changeset.receipt);
+    
+    // TODO check https://eips.ethereum.org/EIPS/eip-658 introduced in Byzantium
+    //verify_receipt(header.receipts_root, header.logs_bloom, receipts_iter)?;
```

Comment:
I added a comment with an explanation, we will probably just skip calculating stateroot in receipts for blocks before Byzantium.

Maybe it will be useful for testing,